ext/em.cpp

```diff
@@ -505,13 +506,9 @@ void EventMachine_t::_DispatchHeartbeats()
 	// is changed out from underneath MyCurrentLoopTime.
 	const EventableDescriptor *head = NULL;
 
-	while (true) {
-		multimap<uint64_t,EventableDescriptor*>::iterator i = Heartbeats.begin();
```

Comment:
This used to run `Heartbeats.begin()` in every loop, and now it just runs it when entering the loop. Are there any negative implications of that change?

Reply:
Good catch! It doesn't look like my code is equivalent. I will put it back as they had it.